### ISSUE TITLE
ERROR with pnpm dlx command

### ISSUE THREAD

**Issue by slozano54** [nvlang/sveltex#9]
Hi !
Thanks for your job.
I actually have difficulty to run the command `pnpm dlx create-sveltex`
I get an error **String.prototype.split called on null or undefined** as yo can see on the screenshot.
![Selection_481](https://github.com/user-attachments/assets/ff62d0a5-ecdb-446f-864e-cfb553df910a)

**Comment by ildella** [comment]
Same with npx and yarn dlx 

```sh
$ npx      create-sveltex
Need to install the following packages:
create-sveltex@0.1.1
Ok to proceed? (y) 

npm warn deprecated inflight@1.0.6: This module is not supported, and leaks memory. Do not use it. Check out lru-cache if you want a good and tested way to coalesce async requests by a key value, which is much more comprehensive and powerful.
npm warn deprecated rimraf@3.0.2: Rimraf versions prior to v4 are no longer supported
npm warn deprecated glob@7.2.3: Glob versions prior to v9 are no longer supported
[ERROR] String.prototype.split called on null or undefined
```

**Comment by nvlang** [comment]
Thanks for the report! I'll try to further diagnose the issue and resolve it this weekend.

**Comment by nvlang** [comment]
I tested this now and it seems the workaround only works with `npx`. My guess would be that `pnpm dlx` and `bunx` don't enforce the `inquirer` version lock properly. I haven't tested it with `yarn dlx`. I'll update the documentation to reflect this.

**Comment by slozano54** [comment]
Thanks for your answer.

In fact, i still have the problem with pnpm.
![Sélection_506](https://github.com/user-attachments/assets/68eaffe1-3428-40ce-b463-c48a3a78cbe4)

Actually I can test your project by addind to an existing svelte project and it's great :)

**Comment by nvlang** [comment]
@slozano54 Thanks for the kind words! I actually just published another update that seems to fix the issue for `pnpm dlx` and `bunx` as well (dfb6a100fd08127ae40463e7819cbf1fd0db096a). In my case I had to run `pnpm dlx create-sveltex@0.1.3` or `bunx create-sveltex@0.1.3` specifically to ensure the right version is run, but for users who don't have an outdated version cached this doesn't seem to be necessary.
